2018-10-Vancouver/agenda.md

```diff
@@ -26,23 +26,23 @@ _We are expecting 300 people to participate_.
+|---------------|--------------|-----------------------------------------------|----------------------------------------|--------------------------------------------|-------------------------------------------|
+| 1:00          | Main         | Intro, Sponsor Talk, Code of Conduct          | @mcollina @dshaw @chowdhurian          |                                            | |
+| 1:10          | Main         | V8 Release Schedule and Bisection             | @hashseed                              | https://github.com/nodejs/summit/issues/100#issuecomment-409152001 | |
+| 1:25          | Main         | The Embedder's POV                            | @codebytere                            | https://github.com/nodejs/summit/issues/100#issuecomment-409629177 | [The Embedder's POV](https://github.com/codebytere/gives-talks/blob/master/node-collab-summit-2018/the_embedders_pov.pdf)|
+| 1:40          | Main         | Automation of Node.js core workflows          | @joyeecheung                           | https://github.com/nodejs/summit/issues/108 | [automation-of-nodejs-workflows](https://docs.google.com/presentation/d/1SZwEOoBxUgaYQLiNk7OAmtvgZ8tFUgdq0IRM6rEAJho/edit?usp=sharing) |
+| 2:00          | Main         | Merger presentation & open forum              | @MylesBorins                           | | |
+| 2:45          |              | Break                                         |                                        | | |
+| 3:15          | Main         | Session presentation, set up the Main room    | @mcollina @dshaw @chowdhurian          | | |
+| 3:30          | Main         | Moderation                                    | @hackygolucky                          | | |
+| 3:30          | Main         | Mentorship                                    | @Bamieh                                | | |
+| 3:30          | Main         | LTS + release                                          |    @MylesBorins          | https://github.com/nodejs/Release/issues/358  | |
+| 3:30          | Main         | Tooling                                          | @boneskull                                       | https://github.com/nodejs/tooling/issues/4 | |
+| 3:30          | Main         | Embedding                                     | @codebytere                       | https://github.com/nodejs/summit/issues/125   | [The Embedder's POV](https://github.com/codebytere/gives-talks/blob/master/node-collab-summit-2018/the_embedders_pov.pdf)|
+| 3:30          | Breakout 1 (Room 118)   | Diagnostics                                   | @kjin , @yunong                   | https://github.com/nodejs/diagnostics/issues/229 | [Diagnostics](https://docs.google.com/a/google.com/presentation/d/e/2PACX-1vS19WYtuxxx6I7_Y7Qy7c4wLARbKEcveYsY-8XgBes0YTVw70j3x22KBltaUGGTNa73TIET6Vvx7Cw7/pub?start=false&loop=false&delayms=3000)|
```

Comment:
@kjin I'm getting "You need permission to access this published document" for the Diagnostics Session slides.